frontend/src/scenes/trends/PersonsModal.tsx

```diff
@@ -102,9 +103,11 @@ export function PersonsModal({
     )
 
     const flaggedInsights = featureFlags[FEATURE_FLAGS.NEW_INSIGHT_COHORTS]
-    // TODO: Re-enable CSV downloads when frontend can support new entity properties
-    // const isDownloadCsvAvailable: boolean = view === InsightType.TRENDS && showModalActions && !!people?.action
-    const isDownloadCsvAvailable: boolean = false
+    const isDownloadCsvAvailable: boolean =
+        !!featureFlags[FEATURE_FLAGS.ASYNC_EXPORT_CSV_FOR_LIVE_EVENTS] &&
+        InsightType.TRENDS &&
+        showModalActions &&
+        !!people?.action
```

Comment:
Generally I say 👍  I wonder though if we couldn't in the future abstract this modal much more nicely to take some sort of `url` parameter which can then automagically be used to load the data relevant to the modal / pagination etc. Would make the CSV exporting much simpler as well then as it simply uses the provided url...